packages/protocol/contracts/L2/TaikoL2.sol

```diff
@@ -24,36 +24,31 @@ import { TaikoL2Signer } from "./TaikoL2Signer.sol";
 contract TaikoL2 is EssentialContract, TaikoL2Signer, ICrossChainSync {
     using LibMath for uint256;
 
+    uint64 public constant GAS_TARGET_PER_L1_BLOCK = 150 * 1e6;
```

Comment:
hmm, should not it be 15 mill ?

Reply:
I was assuming we scale Ethereum by 10 times :) We can adjust this number later. I'll also add a comment.